classes/controller/Controller.php

```diff
@@ -634,9 +634,22 @@ protected function ajaxDie($value = null, $controller = null, $method = null)
             $method = $bt[1]['function'];
         }
 
+        /**
+         * @deprecated deprecated since 1.6.1.1
+         * use 'actionAjaxDieBefore' instead
+         */
         Hook::exec('actionBeforeAjaxDie', array('controller' => $controller, 'method' => $method, 'value' => $value));
+
+        Hook::exec('actionAjaxDieBefore', array('controller' => $controller, 'method' => $method, 'value' => $value));
```

Comment:
You're right, I forgot the hooks aliases.
I will comment the old methods (to let some informations) and add the aliases.

Thank you for your review @PrestaEdit!